### DISCUSSION TITLE
Download one assembly per species:

### DISCUSSION THREAD

**Discussion by EisenRa** [kblin/ncbi-genome-download#186] [Q&A]
Dear authors,

Thank you for writing this tool, it has been a huge help to me!

I have a quick question regarding the downloading of assemblies. I'm attempting to download assemblies based on taxids -- which works great, but I would only like to download one assembly per species. I am aware of the --refseq-category command that will help in this regard (_i.e._ just representative or reference), however, some species do not have 'references' or representative assemblies (_e.g._ https://www.ncbi.nlm.nih.gov/assembly/GCF_003030345.1/ )

Is there some way I can download these even if they don't have references/representatives?

Thanks!
Raphael

**Comment by kblin** [comment]
Hi Raphael,
Unfortunately, there's no straightforward way of doing this. How do you pick which assembly is the one you want if there's a thousand assemblies?
Also keep in mind that the species are occasionally not assigned correctly.

**Comment by tseemann** [comment]
@EisenRa do you mean one of all species, or just *bacterial* species?

**Comment by EisenRa** [comment]
@tseemann only bacterial/archaeal.

**Comment by andrewsanchez** [comment]
@EisenRa You can parse the assembly summary file, group accession IDs together that have the same species tax ID, then feed those IDs to ncbi-genome-download with the -A option.  That would work if you don't care which one you download.
